react/features/invite/functions.js

```diff
@@ -77,3 +78,27 @@ export function inviteRooms(conference, rooms) {
         }
     }
 }
+
+/**
+ * Indicates if an invite option is enabled in the configuration.
+ *
+ * @param {string} name - The name of the option defined in
+ * interfaceConfig.INVITE_OPTIONS.
+ * @returns {boolean} - True to indicate that the given invite option is
+ * enabled, false - otherwise.
+ */
+export function isInviteOptionEnabled(name) {
+    return interfaceConfig.INVITE_OPTIONS.indexOf(name) !== -1;
+}
+
+/**
+ * Get the position of the invite option in the interfaceConfig.INVITE_OPTIONS
+ * list.
+ *
+ * @param {string} optionName - The invite option name.
+ * @private
+ * @returns {number} - The position of the option in the list.
+ */
+export function getInviteOptionPosition(optionName) {
+    return interfaceConfig.INVITE_OPTIONS.indexOf(optionName);
```

Comment:
What's your thought on this: if helper functions are used in one component only, should they put into functions.js?

Reply:
I would agree in general. However, I'm not sure because I want to keep this one together with the isInviteOptionEnabled, which is used in the mapStateToProps, so if I move them in InviteButton, I'll need to put at least this one out of the component so that I can use it in the mapStateToProps. It looks to me cleaner to have them in functions, but it's not a strong opinion.